cmd/root.go

```diff
@@ -22,13 +22,32 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/utils"
 )
 
+// The default log level
+var logLevel = 1
```

Comment:
I'd prefer if we use constants instead of magic numbers; see also the comment about doing this in `logger.go`.

Reply:
+1